docs/quickstarts/packet.md

```diff
@@ -5,262 +5,307 @@
 
-At this point you have access to the Kubernetes cluster and can use it!
-If you don't have Kubernetes experience you can check out the [Kubernetes
-Basics official
-documentation](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/)
-to learn about its usage.
+At this point you should have access to a Lokomotive cluster and can use it to deploy applications.
+
+If you don't have any Kubernetes experience, you can check out the [Kubernetes
+Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/) tutorial.
 
-**Note**: Lokomotive sets up a pretty restrictive Pod Security Policy that
-disallows running containers as root by default, check the [Pod Security Policy
-documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
-for more details.
+>NOTE: Lokomotive uses a relatively restrictive Pod Security Policy by default. This policy
+>disallows running containers as root. Refer to the
+>[Pod Security Policy documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
+>for more details.
 
 ## Cleanup
 
-To destroy the Lokomotive cluster, execute the following command:
+To destroy the cluster, execute the following command:
 
 ```console
-lokoctl cluster destroy --confirm
+lokoctl cluster destroy -v
 ```
 
-You can safely delete the working directory created for this quickstart guide if you no longer
-require it.
+Confirm you want to destroy the cluster by typing `yes` and hitting **Enter**.
 
-## Troubleshooting
+You can now safely delete the directory created for this guide if you no longer need it.
 
-### Stuck at copy controller secrets
+## Troubleshooting
 
-If there is an execution error or no progress beyond the output provided below:
+### Stuck at "copy controller secrets"
 
-```console
+```
 ...
-module.packet-mycluster.null_resource.copy-controller-secrets: Still creating... (8m30s elapsed)
-module.packet-mycluster.null_resource.copy-controller-secrets: Still creating... (8m40s elapsed)
+module.packet-lokomotive-demo.null_resource.copy-controller-secrets: Still creating... (8m30s elapsed)
+module.packet-lokomotive-demo.null_resource.copy-controller-secrets: Still creating... (8m40s elapsed)
 ...
 ```
 
-The error probably happens because the `ssh_pubkeys` provided in the configuration is missing in the
-`ssh-agent`.
-
-To rectify the error, you need to:
+In case the deployment process seems to hang at the `copy-controller-secrets` phase for a long
+time, check the following:
 
-1. Follow the steps [to add the SSH key to the
-   ssh-agent](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent).
-2. Retry [Step 5](#step-5-create-lokomotive-cluster).
+- Verify the correct private SSH key was added to `ssh-agent`.
+- Verify that you can SSH into the created controller node from the machine running `lokoctl`.
```

Comment:
```suggestion
- Verify that you can SSH into the created controller node from the machine running `lokoctl` **without using a password**.
```

I think terraform will only copy the files if you can ssh without issuing any password. If the key is in the agent, that shouldn't be the case, but if you are debugging this, I think it is relevant to point it out. What do you think?

Reply:
AFAIK we don't support password auth for SSH (and I don't think we want to). If you meant something else, please clarify.